gtsfm/utils/io.py

```diff
@@ -116,6 +118,42 @@ def read_bal(file_path: str) -> GtsfmData:
     return gtsfm_data
 
 
+def read_cameras_txt(fpath: str) -> Optional[List[Cal3Bundler]]:
+    """Read camera calibrations from a COLMAP-formatted cameras.txt file.
+
+    Reference: https://colmap.github.io/format.html#cameras-txt
+
+    Args:
+        fpaths: path to cameras.txt file
+
+    Returns:
+        calibrations: calibration object for each camera
+    """
+    if not Path(fpath).exists():
+        return None
+
+    with open(fpath, "r") as f:
+        lines = f.readlines()
+
+    num_cams = int(lines[2].replace('# Number of cameras: ', '').strip())
+    # should have one line per camera
+    assert len(lines[3:]) == num_cams
+    
+    calibrations = []
+    for line in lines[3:]:
+
+        cam_params = line.split()
+        # Note that u0 is px, and v0 is py
+        cam_id, model, img_w, img_h, fx, u0, v0 = cam_params[:7]
+        img_w, img_h, fx, u0, v0 = int(img_w), int(img_h), float(fx), float(u0), float(v0)
```

Comment:
maybe not read image width and height if we dont use it?

Reply:
I think we may want it later; I didn't need it for now, so didn't return it.